CONTRIBUTING.md

```diff
@@ -28,7 +28,14 @@ during their normal contribution hours.
 
 ## Development
 
-This project uses [`tox`](https://tox.readthedocs.io) to automate some aspects
+To quickly get up and running, you can use the `scripts/eachdist.py` tool that
+ships with this project. First create a virtualenv and activate it.
+Then run `python scripts/eachdist.py develop` to install all required packages
```

Comment:
What's the difference between `python scripts/eachdist.py develop` and `python scripts/eachdist.py install --editable`?

Reply:
It is an alias for `install --editable --with-dev-deps --eager-upgrades` (see https://github.com/open-telemetry/opentelemetry-python/blob/cf7e685d7453d8c71bdce23ab744d31b60aa0b41/scripts/eachdist.py#L82)